google-cloud-clients/google-cloud-firestore/src/test/java/com/google/cloud/firestore/conformance/TestDefinition.java

```diff
@@ -23098,7 +23098,7 @@ public int getKindValue() {
     /** <code>.tests.DocChange.Kind kind = 1;</code> */
     public com.google.cloud.firestore.conformance.TestDefinition.DocChange.Kind getKind() {
       com.google.cloud.firestore.conformance.TestDefinition.DocChange.Kind result =
-          com.google.cloud.firestore.conformance.TestDefinition.DocChange.Kind.valueOf(kind_);
+          com.google.cloud.firestore.conformance.TestDefinition.DocChange.Kind.forNumber(kind_);
```

Comment:
Is valueOf() and forNumber() does same thing? From the sound of it, it appears to be different.

Reply:
Yes its doing same if you go inside valueOf method its internally calling forNumber method